frontend/tailwind.config.js

```diff
@@ -6,7 +6,7 @@ const { Z_INDICES } = require("./src/constants/z-indices")
 
 module.exports = {
   content: [
-    "src/**/*.{vue,js,jsx,ts,tsx,mdx}",
+    "./src/**/*.{vue,js,jsx,ts,tsx,mdx}",
```

Comment:
Was something not working when we had no './' here?

Reply:
I don't remember why I added this. Considering that, I'll drop this change.